etc/ci.sh

```diff
@@ -90,9 +105,15 @@ GAPInput
         exit 1
     fi
 
-    bin/gap.sh --cover $COVDIR/${TEST_SUITE}.coverage tst/${TEST_SUITE}.g
+    bin/gap.sh $(coverageFlags "$COVDIR/${TEST_SUITE}.coverage") tst/${TEST_SUITE}.g || returnval=1
 esac;
 
+if ! [ -z "${NO_COVERAGE+x}" ]
+then
+    exit 0
```

Comment:
To repeat the question from my previous review: Shouldn't this be `exit ${retval}` ?